Directory.Build.props

```diff
@@ -9,18 +9,22 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <HighEntropyVA>true</HighEntropyVA>
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>9.0</LangVersion>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <Product>Microsoft Health</Product>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <RepositoryUrl>https://github.com/microsoft/healthcare-components</RepositoryUrl>
     <SdkPackageVersion>3.1.13</SdkPackageVersion>
     <SupportedFrameworks>netcoreapp3.1;net5.0</SupportedFrameworks>
     <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <WarningsAsErrors />
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- CA1031 - Allow Exception to be caught -->
+    <NoWarn>$(NoWarn);CA1031</NoWarn>
```

Comment:
We often catch all exceptions and should probably re-evaluate its usage in some places